bench-jmh/src/main/scala/org/apache/pekko/stream/CollectBenchmark.scala

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
we can't have Lightbend copyright and a standard ASF header - if we need to keep the Lightbend copyright, then we need to use the special ASF header that mentions Akka

Why does this new class need a Lightbend copyright?

Reply:
Becase I copied some code from OPS.scala for benchmark...

Reply:
take out the ASF header that you added and use the ASF header from Ops.scala then